boards/arm/mimxrt1060_evk/mimxrt1060_evk_hyperflash_defconfig

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM=y
+CONFIG_SOC_MIMXRT1062=y
+CONFIG_SOC_SERIES_IMX_RT=y
+CONFIG_BOARD_MIMXRT1060_EVK_HYPERFLASH=y
```

Comment:
I might have missed this in your rt1050 pr, but I don't want to default to external flash just yet. We still need to add support for the boot header for this to be a bootable image.

Reply:
You're right, my mistake. I got `BOARD_MIMXRT1060_EVK_HYPERFLASH` and `CODE_HYPERFLASH` confused